src/sentry/static/sentry/app/utils/streamManager.jsx

```diff
@@ -24,6 +24,8 @@ class StreamManager {
     items = [].concat(items);
     if (items.length === 0) return this;
 
+    items = items.filter((item) => item.hasOwnProperty("id"));
```

Comment:
Were you actually seeing groups that had no IDs? My main concern is how those would exist